contracts/Colony.sol

```diff
@@ -66,6 +66,13 @@ contract Colony is ColonyStorage {
     return token.mint(_wad);
   }
 
+  function mintTokensForColonyNetwork(uint _wad) public {
```

Comment:
Why not just call `mint` on the CLNY token from `ColonyNetwork`?

Reply:
The permission requirements are different, and I thought it would have been less clear if we had to accommodate the default 'auth' modifier, though not impossible. I thought it was clearer to have two separate functions.

`mint` can be called by any authorized user in the colony (i.e. an admin).
`mintTokensForColonyNetwork` can only be called by the `colonyNetwork`, and only on the Common Colony.

We could add `colonyNetwork` to the list of users authorized to call `mint` for the Common Colony, but then there is the possibility that address could be removed from the authority which would break the reputation mining...